app/models/user/tasks/concern.rb

```diff
@@ -0,0 +1,53 @@
+module User
+  module Tasks
+    module Concern
+      extend ActiveSupport::Concern
+
+      included do
+        def agreements
+          @agreements ||= ::User::Tasks::AgreementsFeed.new(uid)
+        end
+
+        def checklist_items
+          @checklist_items ||= ::User::Tasks::ChecklistFeed.new(uid)
+        end
+
+        def web_messages
+          @web_messages ||= ::User::Tasks::WebMessagesFeed.new(uid)
+        end
+
+        def completed_agreements
+          @completed_agreements ||= ::User::Tasks::CompletedAgreements.new(self)
+        end
+
+        def incomplete_agreements
```

Comment:
@developish - Why do you prefix some with `::User::Tasks`, but not others?

Reply:
I think I've gotten into the habit of being over-specific

Reply:
Removed unnecessary prefixes in this file